chess/move_test.go

```diff
@@ -0,0 +1 @@
+package chess
```

Comment:
Is this file necessary? Can we combine a few of these files into a `chess.go`?

Reply:
Removed `move_test.go` along with other unused `*_test.go` files.

I think combining into `chess.go` sounds good after other comments are addressed.

Reply:
Done.